tests/integration/header/header_test.go

```diff
@@ -42,6 +43,28 @@ func (dispatcher) ExecuteTemplate(rw http.ResponseWriter, t safehttp.Template, d
 	return nil
 }
 
+type responseWriter struct {
```

Comment:
nit: `fakeResponseWriter`

Reply:
Just checked other files: feel free to disregard this for consistency. It's not terribly important.